mock/tests/e2e/App.spec.ts

```diff
@@ -54,29 +55,147 @@ test("after I type into the input box, its text changes", async ({ page }) => {
+  await page.getByLabel("Command input").fill("view");
+  await page.getByLabel("Submit button").click();
+  // Assumes that if first two elems of csv were displayed, csv was displayed
+  await expect(page.getByLabel("repl-history")).toContainText("More");
+  await expect(page.getByLabel("repl-history")).toContainText("strings");
+});
+
+test("I cannot try to view a csv before loading", async ({ page }) => {
+  // NOTE: our implementation has the caveat that loading a csv then trying to load a bad second csv and viewing that one will result in the first csv getting displayed
+  //  We do not remove a loaded csv except when overriding it with a new load
+  await page.getByLabel("Login").click();
+  const viewCommand = "view";
+  await page.getByLabel("Command input").fill(viewCommand);
+  await page.getByLabel("Submit button").click();
+  await expect(page.getByLabel("repl-history")).toContainText("No csv loaded");
+});
+
+test("I cannot try to search a csv before loading", async ({ page }) => {
+  await page.getByLabel("Login").click();
+  const searchCommand = "search term";
+  await page.getByLabel("Command input").fill(searchCommand);
+  await page.getByLabel("Submit button").click();
+
+  await expect(page.getByLabel("repl-history")).toContainText("No csv loaded");
+});
+
+test("I cannot load without providing a filepath and a header flag", async ({
+  page,
+}) => {
+  await page.getByLabel("Login").click();
+  const loadCommand = "load incorrect_arguments";
+  await page.getByLabel("Command input").fill(loadCommand);
+  await page.getByLabel("Submit button").click();
+  // header flag is checked first, even before a bad path is checked
+  await expect(page.getByLabel("repl-history")).toContainText(
+    "Missing or incorrect hasHeader argument, please provide the command in the format"
+  );
+});
+
+test("verbose display changes when user enters mode command", async ({
+  page,
+}) => {
+  await page.getByLabel("Login").click();
+  const modeCommand = "mode";
+  await page.getByLabel("Command input").fill(modeCommand);
+  await page.getByLabel("Submit button").click();
+  await page.getByLabel("Submit button").click();
+  // header flag is checked first, even before a bad path is checked
+  await expect(page.getByLabel("repl-history")).toContainText(
+    "Command: Output: No command given"
+  );
+  const loadCommand = "load strings.csv false";
+  await page.getByLabel("Command input").fill(loadCommand);
+  await page.getByLabel("Submit button").click();
+  await expect(page.getByLabel("repl-history")).toContainText(
+    "Command:  Output: No command givenCommand: loadOutput: Loading file at destination strings.csv"
+  );
+  const modeCommand2 = "mode";
+  await page.getByLabel("Command input").fill(modeCommand);
+  await page.getByLabel("Submit button").click();
+  await expect(page.getByLabel("repl-history")).toContainText(
+    "No command givenLoading file at destination strings.csv"
+  );
```

Comment:
We suggest having an integration test where you switch between using the commands mode, load_file, view, and search to see if the brief / verbose ouptus for those commands are correctly displayed and see if there's anything that breaks when switching around different commands.